### PR TITLE
Extend documentation for the minimal tflite example. 

### PR DESCRIPTION
**Pull request by nordstroem**
* Add `tflite::` prefix for all tflite types for consistency.
* Extend documentation of how to read the input/output buffers of the interpreter.

Bonus:
* Adds missing entry for `android_arm64` in `.baselrc`.

### CODE REVIEW

File: tensorflow/lite/examples/minimal/minimal.cc
Comment (talumbau) [inline tensorflow/lite/examples/minimal/minimal.cc:57]: It is curious how the original file is inconsistent on using the `tflite::` namespace. The top of the file has `using namespace tflite;` so my inclination would be either remove that line and use `tflite::` explicitly, or remove the `tflite::` throughout and leave the `using` statement. So, if you prefer to keep the `tflite::` throughout, please remove the `using namespace tflite;` statement on line 31.

**Review by talumbau — CHANGES_REQUESTED** [review]
Thanks for these additions. Just a small comment below. Cheers!

File: tensorflow/lite/examples/minimal/minimal.cc
Comment (johannordstroem) [reply tensorflow/lite/examples/minimal/minimal.cc:55]: I agree! I've removed the using statement now.

Comment (nordstroem) [reply tensorflow/lite/examples/minimal/minimal.cc:55]: I agree! I've removed the `using` statement now.

**Comment by nordstroem** [comment]
Are there any more required changes or what is the status for this PR?

**Comment by gbaned** [comment]
> Are there any more required changes or what is the status for this PR?

@nordstroem  Nothing to do at this moment. This is waiting for reviewer approval.  Thank you!

**Review by talumbau — APPROVED** [review]
Apologies for the delay! Thanks for the change.

**Pull request merged**
